storage/rocksdb/ha_rocksdb.cc

```diff
@@ -19020,6 +19028,22 @@ static bool parse_fault_injection_params(
   return false;
 }
 
+static bool rocksdb_ddse_dict_init(
+    dict_init_mode_t dict_init_mode, [[maybe_unused]] uint version,
```

Comment:
Drop the argument names for fully-unused parameters, use [[maybe_unused]] for the debug build-only parameters only.